MainTel/MainTel-Troubleshooting/MTM2-Troubleshooting/MTM2_recovery.rst

```diff
@@ -0,0 +1,306 @@
+.. Static objects, such as figures, should be stored in the _static directory. Review the _static/README in this procedure's directory on instructions to contribute.
+.. Do not remove the comments that describe each section. They are included to provide guidance to contributors.
+.. Do not remove other content provided in the templates, such as a section. Instead, comment out the content and include comments to explain the situation. For example:
+	- If a section within the template is not needed, comment out the section title and label reference. Include a comment explaining why this is not required.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *Kevin Fanning*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Yiyung Kang, Jacqueline Seron*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _MTM2-Troubleshooting-MTM2-recovery:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+#########################
+MTM2 Recovery
+#########################
+
+
+
+
+.. _MTM2-recovery-Overview:
+
+Overview
+^^^^^^^^
+.. In one or two sentences, explain when this troubleshooting procedure needs to be used. Describe the symptoms that the user sees to use this procedure. 
+
+This procedure should be done when MTM2 goes to fault and exits the Closed Loop status.
+
+This step-by-step guide is the integration of the `Recovery system at night`_ and `Restart control system`_ procedures authored by Te-Wei Tsai. 
+
+.. _`Recovery system at night`:  https://ts-m2gui.lsst.io/error-handling/recover-system-night.html
+
+.. _`Restart control system`: https://ts-m2gui.lsst.io/error-handling/error-handling.html#lsst-ts-m2gui-error-restart-control-system
+
+
+
+.. admonition:: Information
+
+    **MTM2 Possible States** are Idle, TelemetryOnly (disable), Open LoopLoop, Close Loop. Each state is indicated by an integer from 1 to 4.
+
+    You can check the MTM2 state in Chronograf dashboards `M2 status`_, `MTM2`_ or  `LOVE view`_.
+
+    MTM2 should always be in the Close Loop state (state 4). A GIS interlock prevents TMA movement if MTM2 is not in the Close Loop state. 
```

Comment:
*CloseD Loop*, I checked :)

Reply:
all states for changed to "_closed-loop_" I added the '-' because it was the way it was wriiten in the m2 gui documentation.